common/src/Polyhedron_Matcher.h

```diff
@@ -118,36 +122,51 @@ class PolyhedronMatcher {
         return result;
     }
 private:
+    /*
+     * Computes the matching score between the given two faces using the data stored in the vertex relation.
+     *
+     * The matching score between the given faces is the number of all pairs of a vertex of the given left face
+     * and a vertex of the given right face which are also in the vertex relation, unless the faces are identical.
```

Comment:
I still don't quite get the matching scores idea - what does it mean if pair is in the vertex relation?

Reply:
Two vertices (a,b) are in the relation if

- they exist in both the old and the new geometry (unchanged, a.position == b.position)
- a no longer exists, but was connected to a vertex c in the old geometry such that (c,b) are related
- b did not exist in the old geometry, but it is connected to a vertex c in the new geometry such that (a,c) are related

I'll admit it's hard to understand. I'll add more documentation here.